modules/ocf_kubernetes/templates/master/loadbalancer/haproxy.cfg.erb

```diff
@@ -67,6 +77,8 @@ backend kubernetes
   http-request add-header X-Forwarded-Port %[dst_port]
   mode http
   option forwardfor
+  option httpchk OPTIONS / HTTP/1.0
+  http-check expect rstatus (2|3)[0-9][0-9]
```

Comment:
technically we expect exactly 200, but honestly we can probably just leave out this line completely. From https://www.haproxy.com/documentation/aloha/10-0/traffic-management/lb-layer7/health-checks/#check-an-http-service:

> The check is valid if the server answers with a status code of 2xx or 3xx, unless this behavior is changed by the directive http-check expect.

Reply:
I set it to check for exactly `200`, since a `300` should never happen and therefore probably be considered unhealthy?